Gopkg.lock

```diff
@@ -0,0 +1,123 @@
+# This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
+
+
+[[projects]]
+  name = "github.com/cpuguy83/go-md2man"
+  packages = ["md2man"]
+  revision = "a65d4d2de4d5f7c74868dfa9b202a3c8be315aaa"
+  version = "v1.0.6"
```

Comment:
This particular version is actually wrong and problematic.  It pulls in blackfriday v2 and breaks stuff.  Do not approve this PR until this dependency has been updated.